msbuild/Xamarin.Localization.MSBuild/Xamarin.Localization.MSBuild.csproj

```diff
@@ -17,6 +18,61 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="XliffTasks" Version="1.0.0-beta.20154.1" />
+    <EmbeddedResource Update="MSBStrings.resx">
+    </EmbeddedResource>
+    <EmbeddedResource Include="MSBStrings.resx">
+    </EmbeddedResource>

```

Comment:
I don't think you need the Update + Include entries, since nothing is updated, nor nothing new is included:

```suggestion
```

Reply:
I tried removing them, but the include creates the Xamarin.Localization.MSBuild.MSBStrings.resources and without this, the tests fail when trying to get the english dll. Therefore, I think Update can go away but include needs to stay?

Reply:
Or perhaps, I can change the Update for MSBStrings.resx a few lines above these to an Include?